### PR TITLE
Update LiteSubscription with offerPrice prop

### PR DESCRIPTION
**Pull request by bolajiadeyemi**
### Description
Update LiteSubscription with offerPrice prop

### Ticket
https://financialtimes.atlassian.net/browse/UG-414

### Screenshots

| Before | After |
| ------ | ----- |
|    Screenshot 2021-03-19 at 17 45 57    |    
Screenshot 2021-03-19 at 17 47 17

   |

### Reminder
Have you completed these common tasks (remove those that don't apply)?

- [x ] **Stories** updated to use this change
- [x ] **Accessibility** checked for screen readers and contrast
- [x ] **Design Review** ran past the designer
- [x ] **Product Review** ran past the product owner

### CODE REVIEW

File: components/lite-sub-confirmation.jsx [hunk 1]
```diff
@@ -60,7 +61,7 @@ export function LiteSubConfirmation ({
...
-			<p className="ncf__paragraph">For $5 a month, you can now enjoy {isPremium(productCode) ? 'unlimited' : 'standard digital'} access to FT.com, where you can explore not only the product you had purchased, but everything else the FT has to offer. However, if you would like to cancel your subscription, please contact our <a className="barrier__terms-link ncf__link ncf__link--external" href="https://help.ft.com/contact">customer care team</a> and they will arrange this for you.</p>
+			<p className="ncf__paragraph">For {`${offerPrice}`} a month, you can now enjoy {isPremium(productCode) ? 'unlimited' : 'standard digital'} access to FT.com, where you can explore not only the product you had purchased, but everything else the FT has to offer. However, if you would like to cancel your subscription, please contact our <a className="barrier__terms-link ncf__link ncf__link--external" href="https://help.ft.com/contact">customer care team</a> and they will arrange this for you.</p>
```
Comment (jamesnicholls) [inline components/lite-sub-confirmation.jsx:64]: **blocker:** I think we should also make the subscription term (the "a month" text dynamic, as we may have to support other subscription terms, like annual. 

Also I think the price should probably come from the user's subscription, which we should already in the `details` prop. I'll add more information on your [next-subscribe PR](https://github.com/Financial-Times/next-subscribe/pull/1564/files)

**Review by jamesnicholls — CHANGES_REQUESTED** [review]

File: components/lite-sub-confirmation.jsx (hunk 1 above)
Comment (jamesnicholls) [reply components/lite-sub-confirmation.jsx:64]: For the purposes of the smoke test, we are assuming that the only payment term available will be monthly.

**Review by jamesnicholls — APPROVED** [review]

File: components/lite-sub-confirmation.jsx [hunk 2]
```diff
@@ -91,6 +92,7 @@ export function LiteSubConfirmation ({
 
 LiteSubConfirmation.propTypes = {
 	offerName: PropTypes.string.isRequired,
+	offerPrice: PropTypes.string.isRequired,
```
Comment (ludrob) [inline components/lite-sub-confirmation.jsx:95]:
```suggestion
	subscriptionAmount: PropTypes.string.isRequired,
```

File: components/lite-sub-confirmation.jsx [hunk 3]
```diff
@@ -60,7 +61,7 @@ export function LiteSubConfirmation ({
...
-			<p className="ncf__paragraph">For $5 a month, you can now enjoy {isPremium(productCode) ? 'unlimited' : 'standard digital'} access to FT.com, where you can explore not only the product you had purchased, but everything else the FT has to offer. However, if you would like to cancel your subscription, please contact our <a className="barrier__terms-link ncf__link ncf__link--external" href="https://help.ft.com/contact">customer care team</a> and they will arrange this for you.</p>
+			<p className="ncf__paragraph">For {`${subscriptionAmount}`} a month, you can now enjoy {isPremium(productCode) ? 'unlimited' : 'standard digital'} access to FT.com, where you can explore not only the product you had purchased, but everything else the FT has to offer. However, if you would like to cancel your subscription, please contact our <a className="barrier__terms-link ncf__link ncf__link--external" href="https://help.ft.com/contact">customer care team</a> and they will arrange this for you.</p>
```
Comment (jamesnicholls) [inline components/lite-sub-confirmation.jsx:64]:
```suggestion
			<p className="ncf__paragraph">For {subscriptionAmount} a month, you can now enjoy {isPremium(productCode) ? 'unlimited' : 'standard digital'} access to FT.com, where you can explore not only the product you had purchased, but everything else the FT has to offer. However, if you would like to cancel your subscription, please contact our <a className="barrier__terms-link ncf__link ncf__link--external" href="https://help.ft.com/contact">customer care team</a> and they will arrange this for you.</p>
```

**Review by jamesnicholls — COMMENTED** [review]
One minor suggestion.

**Review by mifkata — APPROVED** [review]
That super long line :D

**Pull request merged**